twitch-chat/index.js

```diff
@@ -100,6 +112,9 @@ app.get('/web_words', function(req, res){
   res.sendFile(__dirname + '/web_words.html');
 });
 
+app.get('/config.json', function (req, res) {
```

Comment:
This reveals some secrets, notably the twitch oauth token. So my preference would be to not do this. What need do we have of it?

Reply:
This wont be used by end user. Its for internal use with an upcoming config editor... more on that later...

Reply:
But if i were to remove it, what is your solution to keep the functionality preserved?